Documentation/ProjectJsonToCSProj.md

```diff
@@ -0,0 +1,63 @@
+.NET CLI preview 3 is here, and with it comes the .csproj project file format and msbuild engine. The team has put a lot of focus on making the transition as seamless as possible, but as with any change of this magnitude there are some gotcha's to keep in mind. This issue explains how to use the .NET CLI during this transitional period.
```

Comment:
This looks scary, we could miss so many files in the initial merge. could you do a beyond compare diff with existing CLI to make sure there is no big chuck of code missing?

Reply:
Yes, I did do this.  There were a bunch of files missing from the merge, probably because we are re-merging repos that were previously split from one, and the files had been deleted in one of the repos.